DataProcessing/Program.cs

```diff
@@ -64,12 +64,13 @@ public static void Main()
             try
             {
                 // Run the data downloader/converter.
-                var success = instance.Run();
+                /*var success = instance.Run();
```

Comment:
Please undo this comment.
We need to check if it's a success for both and return exit(1) if one fails
```csharp
var success = instance.Run();
if  (!success)
{
    Log.Error($"QuantConnect.DataProcessing.Program.Main(): Failed to download/process {CoinGeckoUniverseDataDownloader.VendorName} data");
}
if (!instance.GenerateUniverseFiles())
{
    Log.Error($"QuantConnect.DataProcessing.Program.Main(): Failed to generate {CoinGeckoUniverseDataDownloader.VendorName} universe");
   success = false:
}
if (!success) Environment.Exit(1);